src/components/interviews/add.jsx

```diff
@@ -1,16 +1,21 @@
 import React, { useState } from "react";
 import { Link } from "react-router-dom";
-import axios from "axios";
 import DatePicker from "react-datepicker";
- 
-import "react-datepicker/dist/react-datepicker.css";
+import { useSelector, useDispatch } from "react-redux";
+import { addInterview, update } from "../../redux/actions/interview.jsx";
+
 
 function AddInterview() {
-  const [title, setTitle] = useState("");
-  const [start_time, setStartTime] = useState();
-  const [end_time, setEndTime] = useState();
-  const [interviewer_id, setInterviewerId ] = useState();
-  const [interviewee_id, setIntervieweeId ] = useState();
+  let title = useSelector(state => state.interview.title);
```

Comment:
There is a better way. You can return multiple values as an object from useSelector & shallowEqual. Do explore more